rule/use_fmt_print.go

```diff
@@ -0,0 +1,107 @@
+package rule
+
+import (
+	"fmt"
+	"go/ast"
+	"strings"
+
+	"github.com/mgechev/revive/internal/astutils"
+	"github.com/mgechev/revive/lint"
+)
+
+// UseFmtPrintRule lints calls to print and println.
+type UseFmtPrintRule struct{}
+
+// Apply applies the rule to given file.
+func (r *UseFmtPrintRule) Apply(file *lint.File, _ lint.Arguments) []lint.Failure {
```

Comment:
This comment is out of the scope of the PR.

This reminds me something we talked about months ago @alexandear 

The lint.Arguments is now almost, if not totally  useless.

We could add this to a list of things for a v2 (I'm unsure we can do it now)

I remember we also talked about removing goversion dependencies, and mention it as a candidate for a v2.

The v2 is only an idea now, but we could start taking a note of all ideas.

The v2 could be also a moment to move out of code that use deprecated GOROOT thing.

So here it's more a general feedback